9.1.Индексация/9.1.10.py

```diff
@@ -0,0 +1,7 @@
+s = input()
+total = 0
+
+for i in range(0, len(s)):
+    total += int(s[i])
```

Comment:
Лучше просто по символам строки идти:
```suggestion
for c in s:
    total += int(c)
```